index.html

```diff
@@ -10,10 +14,70 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <nav>
+        <img class="logo" src="img/karma-logo.svg" alt="logo" width="35"/>
+        
+        <ul>
+            <li><a href="#" class="active">Meet Karma</a></li>
+            <li><a href="#">How It Works</a></li>
+            <li><a href="#">Store</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#">Login</a></li>
+        </ul>
+        <img id="hamburger" src="img/menu-hamburger.svg" alt=""/>
+    </nav>
+    <header class="hero">
+           
+            <div class="hero-text-and-button">
+                <h1>Introducing Karma</h1>
+                <p>Bring WiFi with you, everywhere you go.</p>
+                <button>Learn More</button>
+            </div>
+    </header>
+  
+    <main>
+        <section>
+            <div>                
+                <h2>Everyone needs a little Karma</h2>
+            </div> 
+            <div class="section-items">
+                <div class="section-item">
+                    <img src="img/icon-devices.svg" alt="" />
```

Comment:
Images should have an alt text -- otherwise someone accessing this on a screen reader would not be able to tell what they are. For example "phone, computer and tablet".